ui/docs/reference/platform/object/download-methods.mdx

```diff
@@ -0,0 +1,72 @@
+---
+order: 3
+title: Download Methods
+---
+# Download Methods
+
+## Introduction
+
+The Object Service does not directly handle object content, but acts as an
+intermediary on behalf of other data storage service providers. That makes
```

Comment:
could it be called a "directory service to other data storage providers" or something like that? Intermediaries still could be handling the bits in my reading of this.